regions/core/core.py

```diff
@@ -178,10 +179,14 @@ def plot(self, ax=None, **kwargs):
 
         Parameters
         ----------
-        ax : `~matplolib.axes`, optional
+        ax : `~matplotlib.axes`, optional
             Axis
         """
-        patch = self.make_patch(ax, **kwargs)
+        import matplotlib.pyplot as plt
+
+        ax = plt.gca() if ax is None else ax
```

Comment:
Personally I prefer when even the simplest if/else-s are written into multi lines, and here you don't even need the else part.